rust/kcl-lib/tests/assembly_mixed_units_cubes/artifact_commands.snap

```diff
@@ -124,8 +84,8 @@ description: Artifact commands assembly_mixed_units_cubes.kcl
       "type": "move_path_pen",
       "path": "[uuid]",
       "to": {
-        "x": -10.0,
-        "y": -10.0,
+        "x": -254.0,
+        "y": -254.0,
```

Comment:
Are the coordinate changes here intentional?

Reply:
Yes, they're inches to mm